projects/go-lib/src/lib/components/go-timepicker/go-time.ts

```diff
@@ -0,0 +1,30 @@
+import { Subject } from 'rxjs';
+
+export class GoTime {
```

Comment:
What's the purpose of having this class to handle open and close and the selectedTime instead of handling that as a top down approach from the go-time component?

Reply:
Yes. I agree with you. I have written some code and remove that GoTime class. It's working fine now. Please check those changes.